lib/github_commit_graph/github_commit_graph.rb

```diff
@@ -77,7 +77,11 @@ def longest_streak(days)
 
     def current_streak(days)
       streak = 0
-      days.reverse_each do |day|
+      
+      # The current day can count towards the streak but it shouldn't break the streak
+      streak += 1 if days.last['contributionCount'].positive?
```

Comment:
```suggestion
      streak += 1 if days.last[:contributionCount].positive?
```

(in this context the keys are symbolized, so string lookup will yield `nil`, which doesn't respond to `positive?`)